api/Cargo.toml

```diff
@@ -41,3 +41,8 @@ uuid = { version = "1.16.0", features = ["serde", "v4", "v7"] }
 validator = { version = "0.20.0", features = ["derive"] }
 base32 = "0.5.1"
 axum-prometheus = "0.8.0"
+
+[dev-dependencies]
+test-context = "*"
+axum-test = "*"
+sea-orm = { version = "1.1.14", features = ["sqlx-postgres"]}
```

Comment:
this is only really needed to interact with the db on the tests, e.g. for white box testing. There is a test below showing how to. sqlx is also an option here, I am just less familiar with sqlx, so if someone could refactor into sqlx, also good for me